docs/kubernetes-installers.md

```diff
@@ -39,7 +39,34 @@ want. Once the cluster has been created, this CIDR cannot be changed.
 
 ### Rancher
 
-When creating a workload cluster, set the [network
+Follow these [steps](https://ranchermanager.docs.rancher.com/pages-for-subheaders/rancher-on-a-single-node-with-docker) to deploy Rancher server UI dashboard.
+
+* Sign in to the UI dashboard, and switch to cluster management.
+
+<img src="https://downloads.antrea.io/static/04062023/rancher_cluster_management.png" width="800">
+
+* Create new Node template there under the RKE configuration section.
+
+<img src="https://downloads.antrea.io/static/04062023/rancher_node_template.png" width="800">
+
+* After saving the template, proceed with creating new cluster, click on create button under the clusters tab, then enter the name, number of worker and control plane Nodes respectively.
+
+<img src="https://downloads.antrea.io/static/04062023/rancher_create.png" width="800">
+
+* click on "Edit as YAML", and set the network plugin to null, and add an addon for Antrea, in the following manner:
+addons_include:
+
+  -link of the antrea.yml file
```

Comment:
updated

Reply:
@jainpulkit22 this is not addressed, have you pushed the latest change?

Reply:
no, i was still waiting for the final call that whether do I need to create a separate doc for rancher or not.

Reply:
OK, you don't have to reply "updated" if the latest codes are not pushed, otherwise, it will confuse reviewers. I think you won't need a new doc as long as you describe clearly about Antrea installation part. As @antoninbas  point out, our focus should be Antrea.
Please check with @antoninbas and @tnqn offline to get the final conclusion if you are not so sure. We'd better to merge this in v1.13.